src/sklearn_evaluation/plot/regression.py

```diff
@@ -0,0 +1,122 @@
+"""
+Plotting functions for regression plots.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from sklearn_evaluation.telemetry import SKLearnEvaluationLogger
+from sklearn.linear_model import LinearRegression
+
+
+@SKLearnEvaluationLogger.log(feature='plot')
+def residual(y_true, y_pred, ax=None):
+    """
+    Plot the residuals between measured and predicted values.
+
+    Parameters
+    ----------
+    y_true : array-like, shape = [n_samples]
+        Measured target values (ground truth).
+    y_pred : array-like, shape = [n_samples]
+        Predicted target values.
+    ax : matplotlib Axes
+        Axes object to draw the plot onto, otherwise uses current Axes
+
+    Returns
+    -------
+    ax: matplotlib Axes
+        Axes containing the plot
+
+    Examples
+    --------
+    .. plot:: ../../examples/residual.py
+
+    """
+
+    _check_parameter_validity(y_true, y_pred)
+
+    if ax is None:
+        ax = plt.gca()
+
+    # horizontal line for residual=0
+    ax.axhline(y=0)
+
+    ax.scatter(y_pred, y_true-y_pred)
+
+    _set_ax_settings(ax, 'Predicted Value', 'Residuals', 'Residuals Plot')
+    return ax
+
+@SKLearnEvaluationLogger.log(feature='plot')
+def prediction_error(y_true,
+                     y_pred,
+                     model=LinearRegression(),
```

Comment:
update this to `model=None` since it's a bad practice to have objects as defaults. in the function you can do:

```python
model = model or LinearRegression()
```

this way, the `model` variable will take the user provided value, and if None, it'll take the `LinearRegression()`